components/tree-select/index.tsx

```diff
@@ -38,11 +38,11 @@ export interface TreeSelectProps<T>
 class TreeSelect<T> extends React.Component<TreeSelectProps<T>, {}> {
   static TreeNode = TreeNode;
 
-  static SHOW_ALL = SHOW_ALL;
+  static SHOW_ALL: typeof SHOW_ALL = SHOW_ALL;
```

Comment:
Use `readonly`?

Reply:
```suggestion
  static readonly SHOW_ALL = SHOW_ALL;
```

Reply:
Thanks, I'm not familiar with ts.

Fixed.